locale/cs/LC_MESSAGES/django.po

```diff
@@ -3206,7 +3206,7 @@ msgid ""
 "Our volunteers haven't translated this article into <bdi>%(locale)s</bdi> "
 "yet. <a href=\"%(help_link)s\">Join us and help get the job done!</a>"
 msgstr ""
-"Naši dobrovolníci ještě tento článek do jazyka <bdi>%s (locale)</bdi> "
+"Naši dobrovolníci ještě tento článek do jazyka <bdi>%(locale)s</bdi> "
```

Comment:
@jwhitlock Thanks for spotting the issue with referencing strings directly!

It should be fixed now:
https://pontoon.mozilla.org/cs/mdn/LC_MESSAGES/django.po/?string=141905